guardian/bonobo
# Add email verification

## What does this change?

This updates the developer sign up flow to include an email verification step. Details can be found [here.](https://docs.google.com/document/d/1EThtYQigLa24xoaE4Niec-3iJs5xpDxLDwT5OzjhSak/edit)

## How to test

Deploy to CODE and enable emails in config - you should be able to test the whole sign up flow end-to-end.

## How can we measure success?

People with fake email addresses can't obtain API keys any more.

## Have we considered potential risks?

This is a medium risk change. It introduces potentially breaking changes to the sign up logic, and includes storage changes, UI changes, and more. It has been tested end-to-end but we should keep and eye on it after it goes out.

## Images

In document attached above.
